Core/Src/main.c

```diff
@@ -19,6 +19,9 @@
 /* USER CODE END Header */
 /* Includes ------------------------------------------------------------------*/
 #include "main.h"
+#include "adc.h"
```

Comment:
This is for the in-microcontroller ADC, for this bootcamp only a separate physical ADC chip.